data/SimonVerhoeven.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Simon Verhoeven",
+  "type": "public",
+  "bio": "I'm a software developer living in Belgium that's always open for an interesting chat.ob",
+  "tags": [
+    "OpenSource",
+    "Javascript",
+    "Java",
+    "Angular",
+    "SQL",
+    "Docker"
+  ],
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/simon_verhoeven",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Mastodon",
+      "url": "https://twitter.com/simon_verhoeven",
+      "icon": "FaMastodon"
+    },
+    {
+      "name": "LinkedIn: let's connect",
+      "url": "https://www.linkedin.com/in/simonverhoeven/",
+      "icon": "FaTwitter"
```

Comment:
here you put your linkenIn link but you add twitter icon. please correct it to `FaLinkedin` in line 27

Reply:
Thank you